docs/docs/configuration/overview.md

```diff
@@ -264,6 +264,14 @@ Available variables for standard logging:
 | File | main.go:40 | The file and line number of the logging statement. |
 | Message | HTTP: listening on 127.0.0.1:4180 | The details of the log statement. |
 
+## Example configuration for json logging
+
+```cfg
+auth_logging_format = '{"client": "{{.Client}}", "username": "{{.Username}}", "timestamp": "{{.Timestamp}}", "status": {{.Status}}, "message": "{{.Message}}"}'
+request_logging_format = '{"client": "{{.Client}}", "username": "{{.Username}}", "timestamp": "{{.Timestamp}}", "host": "{{.Host}}", "request_method": "{{.RequestMethod}}", "upstream": "{{.Upstream}}", "request_uri": {{.RequestURI}}, "protocol": "{{.Protocol}}", "user_agent": {{.UserAgent}}, "status_code": {{.StatusCode}}, "response_size": {{.ResponseSize}}, "request_duration": {{.RequestDuration}}}'
```

Comment:
It might be worth adding a note below this that the `{{.UserAgent}}` and `{{.RequestURI}}` templates are already quoted (hence not quoted in this sample correctly).

I've definitely made that mistake making JSON templates for my logs too 😅 

That way it is clear & the intention isn't missed in the middle of all that template syntax that is hard to read in a row.